packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -29,6 +29,8 @@ enum Artifact {
   dart2jsSnapshot,
   kernelWorkerSnapshot,
   flutterWebSdk,
+  devFinder,
+  fuchsiaSshConfig,
```

Comment:
This is not really an artifact, and more of a device specific config. I'll look into how it is generated so it can be included as part of the external workflow.